test/e2e/test-always-on.ts

```diff
@@ -43,15 +42,15 @@ module.exports = (testOrchestrator, testConfig) => {
     console.log(stats)
   })
 
-  test('test with simple zome call via instance', async t => {
+  test('test with simple zome call via cell', async t => {
     t.plan(3)
     const C = testConfig()
     const orchestrator = await testOrchestrator()
     orchestrator.registerScenario('simple zome call', async s => {
-      const players = await s.players({ alice: C.alice }, true)
+      const players = await s.players({ alice: C.players.alice }, C.initialization)
       const { alice } = players
-      const instance = alice.instance('app')
-      const hash = await instance.call('main', 'commit_entry', { content: 'content' }).then(x => x.Ok)
+      const cell = alice.cell('app:cell')
```

Comment:
I was wondering where this was coming from and then noticed that this test is disabled.

Is there an example anywhere of creating a `Cell` and making calls using that? It would be good to test that. I suspect you would need two params to this though, the app id and the cell nick.

Reply:
this is part of an outdated API. sorry you even had to look at it. 

Creating `Cell`s and calling those is the main/only deal